assets/src/edit-story/app/layout/index.js

```diff
@@ -18,6 +18,7 @@
  * External dependencies
```

Comment:
Changes in this file are a bit unrelated, but contribute to less content shifting in the design panel by having the scroll bar be absolutely positioned.